tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/jvmmodel/TclCoercionComputer.xtend

```diff
@@ -35,15 +44,65 @@ class TclCoercionComputer {
 		typeReferenceUtil.initWith(resourceSet)
 	}
 
+	def boolean isCoercionPossible(JvmTypeReference targetType, JvmTypeReference sourceType, String value) {
```

Comment:
Why are there 3 public methods all called `isCoercionPossible`? Do we need them all public?